css/components/alert.css

```diff
@@ -0,0 +1,59 @@
+/* === simple alert ==== */
+
+.alert{
+    width: 100%;
+    padding: .7rem .7rem;
```

Comment:
`padding: .7rem .7rem;` is same as `padding:.7rem;`, so you can replace that.

Reply:
thanks for pointing out.